docker/development/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN git clone https://github.com/geographiclib/geographiclib /tmp/geographiclib
 
 ## Open Space Toolkit ▸ Core
 
-ARG OSTK_CORE_VERSION="0.6.0"
+ARG OSTK_CORE_VERSION="0.7.0"
```

Comment:
All good to use this? If so also update python bindings requirements.txt file

Reply:
thanks for this, done!

Reply:
Added a explicit dependency.

Reply:
cc @vishwa2710